docs/core/integration-quality-scale/rules/iqs048.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "IQS048 - The documentation described the provided service actions that can be used"
+---
+
+IQS048 is as follows:
+
+The documentation described the provided service actions that can be used
+
+## Reasoning
+
+Integrations can register service actions to provide functionality that is not possible with normal entities.
+These service actions can be harder to use, so we want to make sure that the documentation describes both what they do, and what the parameters are.
+
+## Example implementation
+
+```markdown
+## Actions
+
+The integration provides the following actions.
+
+### Get schedule
+
+The `my_integration.get_schedule` service is used to fetch a schedule from the integration.
+
+| Data attribute         | Optional | Description                                          |
+|------------------------|----------|------------------------------------------------------|
+| `config_entry_id`      | No       | The ID of the config entry to get the schedule from. |
```

Comment:
```suggestion
The action supports the following data attributes: 

- `config_entry_id`: The ID of the config entry to get the schedule from. Not optional.
```
One possibility would be to use a list instead. If there are more columns, or columns that are not obvious, the titles could be mentioned in the introductory paragraph, so that readers know what they are looking at.

Reply:
![CleanShot 2024-11-18 at 13 39 06@2x](https://github.com/user-attachments/assets/d24e92e9-4939-4e1c-a2c2-b0a78bd22704)

We should not add the technical identifier in the heading, it should describe what it does and not show an internal 😬